modules/nf-core/truvari/consistency/main.nf

```diff
@@ -0,0 +1,54 @@
+
+process TRUVARI_CONSISTENCY {
+    tag "$meta.id"
+    label 'process_single'
+
+    conda "${moduleDir}/environment.yml"
+    container "${ workflow.containerEngine == 'singularity' && !task.ext.singularity_pull_docker_container ?
+        'https://depot.galaxyproject.org/singularity/truvari:4.1.0--pyhdfd78af_0':
+        'biocontainers/truvari:4.1.0--pyhdfd78af_0' }"
+
+    input:
+    tuple val(meta), path(vcf1), path(vcf2), path(vcf3)
```

Comment:
Hi I've checked the tool out while reviewing the second time and noticed that it's possible to supply as many VCFs as you can. Can you please change the input to account for this? Something like this should work:
```
tuple val(meta), path(vcfs)
```

Where `vcfs` is a tuple of many VCFs

Reply:
well I thought it could be nonconventional to use many vcfs but I tried and it seems to be working for at least 4. So I changed it now.

Reply:
Should we finalize this? @nvnieuwk

Reply:
Yeah I think it's better to have the option and not use it then having to use it, but at the cost of changing the modules again. And sorry I took some days off, I'll review now